keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/PersistentEvent.kt

```diff
@@ -13,8 +13,8 @@ import java.time.Instant
   include = JsonTypeInfo.As.PROPERTY
 )
 @JsonSubTypes(
-  JsonSubTypes.Type(value = ApplicationEvent::class, name = "application"),
-  JsonSubTypes.Type(value = ResourceEvent::class, name = "resource")
+  JsonSubTypes.Type(value = ApplicationEvent::class),
+  JsonSubTypes.Type(value = ResourceEvent::class)
```

Comment:
Does this change have an impact on the resulting JSON?

Reply:
No. I always have to go back and look at the docs for this, but the `name` parameter is used is the _"logical type name used as the type identifier for the class"_, which, for abstract classes and interfaces, never ends up serialized anywhere. In other words, this only really matters in concrete objects.

Reply:
ah, yeah. Missed that this was abstract. Of course.